### DISCUSSION TITLE
Png files randomly broken on upload

### DISCUSSION THREAD

**Discussion by xl3lackout** [community/community#22030] [Other Feature Feedback, Questions, & Ideas]
<p>Ive been using github pages for a few months now doing a translation project here: <a href="https://xl3lackout.github.io/MHFZ-Ferias-English-Project/" class="inline-onebox" rel="noopener nofollow ugc">Ferias English</a>, but I mass uploaded some files yesterday, and now no .png file on the site works. Ive deleted and recommited them plenty of times, and while it displays the files correctly on github desktop, once pushed and built they break. All jpeg files are fine, im not too sure what caused it.</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>I notice your repository has a <code>.gitattibutes</code> file that marks <em>all</em> files as text.</p><aside class="onebox githubblob" data-onebox-src="https://github.com/xl3lackout/MHFZ-Ferias-English-Project/blob/c434de9c49a425df26153dbb355ad475c9042009/.gitattributes">
  <header class="source">

      <a href="https://github.com/xl3lackout/MHFZ-Ferias-English-Project/blob/c434de9c49a425df26153dbb355ad475c9042009/.gitattributes" target="_blank" rel="noopener">github.com</a>
  </header>

  <article class="onebox-body">
    <h4><a href="https://github.com/xl3lackout/MHFZ-Ferias-English-Project/blob/c434de9c49a425df26153dbb355ad475c9042009/.gitattributes" target="_blank" rel="noopener">xl3lackout/MHFZ-Ferias-English-Project/blob/c434de9c49a425df26153dbb355ad475c9042009/.gitattributes</a></h4>


      <pre><code class="lang-gitattributes"># Auto detect text files and perform LF normalization
* text working-tree-encoding=UTF-8</code></pre>




  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p>Combined with enabling LF normalization that will mess with any modified binary files, like PNGs. You should limit the text modifications to actual text files (e.g. <code>*.htm</code>, from looking at your repository).</p>

**Comment by xl3lackout** [comment]
<p>Yeah that was it. I changed the attributes not really knowing what i was doing, in attempts to fix a problem i was having with the files being Shift-JIS encoded. I didnt suspect it was a problem but i guess overwriting the image files caused the oversight to suddenly cause problems. Thanks for your help.</p>
